de-lib/src/main/java/org/iplantc/de/desktop/client/views/windows/DeDiskResourceWindow.java

```diff
@@ -35,18 +38,23 @@
 public class DeDiskResourceWindow extends IplantWindowBase implements FolderSelectionEvent.FolderSelectionEventHandler {
 
     public static final String DATA = "#data";
+    public static final String DE_DATA_DETAILSPANEL_COLLAPSE = "de.data.detailspanel.collapse#";
+
+
     private final DiskResourcePresenterFactory presenterFactory;
     private final IplantDisplayStrings displayStrings;
     private DiskResourceView.Presenter presenter;
 
-
     @Inject
     DeDiskResourceWindow(final DiskResourcePresenterFactory presenterFactory,
-                         final IplantDisplayStrings displayStrings) {
+                         final IplantDisplayStrings displayStrings, final  UserInfo userInfo) {
         this.presenterFactory = presenterFactory;
         this.displayStrings = displayStrings;
+        this.userInfo = userInfo;
         setHeading(displayStrings.data());
-        setSize("900", "480");
+        String width = getSavedWidth(WindowType.DATA.toString());
+        String height = getSavedHeight(WindowType.DATA.toString());
+        setSize((width == null) ? "820" : width, (height == null) ? "400" : height);
         setMinWidth(900);
         setMinHeight(480);
```

Comment:
Not required for this PR, but this is probably a good time to move all of our window sizes to appearance classes.  

Also, for this change in particular I'm not sure what will happen if we set the size to one thing, but then set the minimum sizes to something greater than what was initially set?

Reply:
I think I've forced this situation in testing before, by opening a window in a really small browser. The window opens smaller than the min settings, but as soon as you try to resize it, it snaps to the minimum sizes and forces the user to only increase the size from there.